### ISSUE TITLE
Java 9 support

### ISSUE THREAD

**Issue by Ciruman** [codecentric/javafxsvg#7]
Hi,

I like this library but I was wondering what will happen with it in Java 9? 
Internally it uses `com.sun.javafx.iio.`

Thanks,
Diego

**Comment by 0x4a616e** [comment]
Right, that indeed is a big issue. Unfortunately, I haven't been able yet to test everything with JavaFX 9 and jigsaw enabled. If that fails and there will be no alternative way to set a new ImageLoaderFactory in place the only (unsatisfying) remaining option will probably be to compile everything with `--add-exports`. But I haven't given up on a jigsaw compliant solution yet.

**Comment by Ciruman** [comment]
Ok. Thank you.

**Comment by jiggersplat** [comment]
Any progress on this?

**Comment by FireController1847** [comment]
@0x4a616e Any alternative to this or status update?

**Comment by 0x4a616e** [comment]
I'm sad to say that even with javafx11, there is no way to set a custom image loader factory using public api... So i'm afraid that `--add-exports` is really all I have right now :/

**Comment by FireController1847** [comment]
That's really unfortunate to hear. Oh well, I'll stay starred to this if an alternative comes in the future :)

**Comment by cervonwong** [comment]
Is there any examples on how to use this library with JDK 8+ ?

**Comment by 0x4a616e** [comment]
All the examples in the readme or the test package are for JDK 8

**Comment by xeruf** [comment]
For gradle kotlin dsl:
```kt
tasks {
    run.configure {
        jvmArgs = listOf("--add-exports=javafx.graphics/com.sun.javafx.iio=ALL-UNNAMED", "--add-exports=javafx.graphics/com.sun.javafx.iio.common=ALL-UNNAMED")
    }
}
```
Working with OpenJDK 11+OpenJFX
